python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,876 @@
+    "\n",
+    "Additionally from 2010 on there were imports for [Surrey, BC](https://wiki.openstreetmap.org/wiki/Vancouver/Imports/Surrey) which might be related as this city is part of the Metro Vancouver Region which was chosen as spatial boundary of this analysis according to local mapping community generally lots of data from aerial imagery data jump around the beginning of 2013 and relatively high though not unusual user activity.  \n",
+    "\n",
+    "#### Ottawa:    \n",
+    "\n",
+    "In Ottawa there are three times of minor user activity until 2009 and subsequently an increasing trend until 2018 with times of lower user activity between 2014 and Mid 2015. The local mapping community also [mentions](https://wiki.openstreetmap.org/wiki/Ottawa#Status) that most of the info is from CanVac data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "liable-victim",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>region</th>\n",
+       "      <th>mean</th>\n",
+       "      <th>median</th>\n",
+       "      <th>min</th>\n",
+       "      <th>max</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Vancouver</td>\n",
+       "      <td>7.191083</td>\n",
+       "      <td>6.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>25.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Halifax</td>\n",
+       "      <td>2.878981</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>13.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Edmonton</td>\n",
+       "      <td>1.719745</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>7.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Ottawa</td>\n",
+       "      <td>3.681529</td>\n",
+       "      <td>3.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>13.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      region      mean  median  min   max\n",
```

Comment:
I'm always having some troubles with float values representing persons in statistics, because you can't have like 7.19 active users :D 
Maybe we can just mention the rounded mean values in the text then?

Reply:
I also think we don't need 3 ways of presenting the same data, so through a table, a box plot and a bar chart diagram. I would suggest to show just one of the graphs, whichever you would prefer.

Reply:
yeah I've been thinking about that too, i'll reduce it